packages/KBadge/KBadge.vue

```diff
@@ -49,14 +49,17 @@ export default {
 @import '~@kongponents/styles/_variables.scss';
 
 .k-badge {
-  display: inline-flex;
+  display: inline-block;
```

Comment:
You can remove justification as it won't work in block

Reply:
oh thought I removed that thanks!